code/modules/client/preference_setup/loadout/lists/utility.dm

```diff
@@ -37,6 +37,11 @@
 	path = /obj/item/modular_computer/tablet/preset/custom_loadout/cheap
 	cost = 2
 
+/obj/structure/bed/chair/wheelchair
+	display_name = "wheelchair"
+	pathc = /obj/structure/bed/chair/wheelchair
```

Comment:
/obj/structure/bed/chair/wheelchair points to  /obj/structure/bed/chair/wheelchair, well it won't compile.
You misspelled pathc for path

Reply:
thank you i am very dumb and can only webedit on linux